kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/resources/operator/KroxyliciousOperatorBundleInstaller.java

```diff
@@ -138,7 +138,8 @@ private void applyDeploymentFile() {
                 .endMetadata()
                 .editSpec()
                 .editFirstContainer()
-                .withImage(ImageUtils.changeRegistryOrgAndTag(deploymentImage, Environment.KROXY_REGISTRY, Environment.KROXY_ORG, Environment.KROXY_TAG))
+                .withImage(TestUtils.changeRegistryOrgImageAndTag(deploymentImage, Environment.KROXY_REGISTRY,
```

Comment:
I think it would be better if the operator and operand image were fully independent of one and other.  Here you are building the operator image from parts of the operand image url.

Reply:
I suppose that both operator and operand images would be in the same repository, but if that couldn't be the case we should split it yes

Reply:
It is better if we split them.  As a developer you want the flexibility.  You might be testing a home spun operator with a published operand.